analytics/logger.go

```diff
@@ -0,0 +1,36 @@
+package analytics
+
+import "github.com/bitrise-io/bitrise/log"
+
+type legacyLogger struct {
+	debug bool
+	log.Logger
```

Comment:
The legacyLogger uses the global log functions from the log package. Can you tell me why this `log.Logger` variable is necessary here because as I understand it it will not be used.

Reply:
log.Logger implements the remaining methods defined by the `go-utils/v2/log.Logger` interface.
This way, we extend the new (`bitrise-io/log.Logger`) with the missing methods, so that `legacyLogger` can be passed to the [tracker constructor](https://github.com/bitrise-io/bitrise/pull/818/files#diff-ee3fd0c2a79d62b7da48113df24fa0c13b0d3e8db9cdc7557433f4ae63c13115R123-R132).